packages/connect/src/data/coinInfo.ts

```diff
@@ -13,19 +13,8 @@ const bitcoinNetworks: BitcoinNetworkInfo[] = [];
 const ethereumNetworks: EthereumNetworkInfo[] = [];
 const miscNetworks: MiscNetworkInfo[] = [];
 
-// TODO: replace by structuredClone() after updating TS
-export function cloneCoinInfo<T>(info: T): T {
-    const jsonString = JSON.stringify(info);
-    if (jsonString === undefined) {
-        // jsonString === undefined IF and only IF obj === undefined
-        // therefore no need to clone
-        return info;
-    }
-    return JSON.parse(jsonString);
-}
-
 export const getBitcoinNetwork = (pathOrName: number[] | string) => {
-    const networks = cloneCoinInfo(bitcoinNetworks);
+    const networks: BitcoinNetworkInfo[] = window.structuredClone(bitcoinNetworks);
```

Comment:
It should be necessary to use window and we should avoid that, because it will break code in other envs like RN or Node

Reply:
yes, this is a no go in connect-code. window is only allowed in connect-web package. I thinking if we could improve tooling to avoid this next time?

Reply:
tests caught that as expected https://github.com/trezor/trezor-suite/runs/7961088382?check_suite_focus=true#step:7:227 still would be great to improve tsconfig to forbit using of window in ths project

Reply:
I apologize, of course -_- 13a81ecd6d1bc8eea0bbe9f5132ba46dd6256e5b

Reply:
@mroz22 maybe eslint will be better for this kind of stuff